### ISSUE TITLE
Cannot build on Mac OS X without X11

### ISSUE THREAD

**Issue by Peter2121** [LibVNC/x11vnc#70]
As mentioned in docs, I used `./configure --without-x` to configure, but the build is failed with the error:
```
  CC       x11vnc-xi2_devices.o
xi2_devices.c:26:10: fatal error: 'X11/Xproto.h' file not found
#include <X11/Xproto.h> 
```
I have no X11 installed, according to docs I can build x11vnc without it. How should I proceed?

**Comment by bk138** [comment]
I guess you are using version...?

**Comment by Peter2121** [comment]
I use the version from GIT (`git clone...`) on Mac OS X 10.13.3

**Comment by Peter2121** [comment]
If I give him dummy X11 headers - it passes compile stage, but cannot link anything - `Undefined symbols` for all libraries (it seems that no library is included in link command). It seems that MacOS X build is broken.

**Comment by bk138** [comment]
Not exactly: https://travis-ci.org/LibVNC/x11vnc 

It's very possible that building without X is broken though.

**Comment by Peter2121** [comment]
I don't see in your travis build any traces of libjpeg. Do you test without it? 
As about openssl - brew installs it in `/usr/local/Cellar/openssl/x.y.z`, but you use `/usr/local/opt/openssl` in travis build. Why don't you use brew as source of openssl? Is it supported?
BTW, it seems that `configure` script does not understand `--with-jpeg` argument...

**Comment by bk138** [comment]
Closed by 64f4976fff7804a3bb1566327901788cf0dce95c.
